src/libNode/FinalBlockProcessing.cpp

```diff
@@ -716,7 +716,10 @@ bool Node::ProcessFinalBlockCore(const bytes& message, unsigned int offset,
       LOG_GENERAL(WARNING, "StoreFinalBlock failed!");
       return false;
     }
-    if (!LOOKUP_NODE_MODE) {
+    // if lookup and loaded microblocks, then skip
```

Comment:
Do you need to acquire `m_mutexUnavailableMicroBlocks`?

Reply:
fixed